Form/ChoiceList/ModelChoiceList.php

```diff
@@ -20,7 +20,18 @@
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Symfony\Component\PropertyAccess\PropertyPath;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\ModelChoiceList class is deprecated since SF 2.7'.
```

Comment:
Are you sure that this will trigger useless deprecation notices? AFAIK, the file is included by the autoloader only when you use the class defined in it.

Deprecations are also triggered this way in Symfony repository (c.f. https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/Extension/Core/ChoiceList/SimpleChoiceList.php#L14).